### DISCUSSION TITLE
Pinball Space Cadet stopped working!

### DISCUSSION THREAD

**Discussion by azgeek** [community/community#76197] [Packages]
### Select Topic Area

Bug

### Body

$ /usr/bin/flatpak run --branch=stable --arch=x86_64 --command=SpaceCadetPinball com.github.k4zmu2a.spacecadetpinball
Game version: 2.1.0
Command line: 
Compiled with: SDL 2.28.4; SDL_mixer 2.6.3; ImGui 1.89.2 WIP HW
*** stack smashing detected ***: terminated

$ uname -a
Linux cosmo 4.15.0-213-generic #224-Ubuntu SMP Mon Jun 19 13:30:12 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

$ cat /etc/issue
Linux Mint 19.3 Tricia \n \l

**Comment by azgeek** [comment]
Help please! I am so depressed!

**Comment by Akash1134** [comment]
Hi @azgeek ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
